src/browser/modules/D3Visualization/graphStyle.ts

```diff
@@ -23,544 +23,535 @@ import {
-    }
+  defaultIconCodes = function(): DefaultIconCodeType[] {
+    return defaultIconCodes
+  }
 
-    GraphStyle.prototype.defaultColors = function() {
-      return defaultColors
-    }
+  defaultArrayWidths = function(): DefaultArrayWidthType[] {
+    return defaultArrayWidths
+  }
 
-    GraphStyle.prototype.interpolate = function(str: any, item: any) {
-      let ips = str.replace(/\{([^{}]*)\}/g, (_a: any, b: any) => {
-        const r = item.propertyMap[b]
-        if (typeof r === 'object') {
-          return r.join(', ')
-        }
-        if (typeof r === 'string' || typeof r === 'number') {
-          return r
-        }
-        return ''
-      })
-      if (ips.length < 1 && str === '{type}' && item.isRelationship) {
-        ips = '<type>'
+  defaultColors = function(): DefaultColorType[] {
+    return defaultColors
+  }
+
+  interpolate = (str: any, item: any) => {
+    let ips = str.replace(/\{([^{}]*)\}/g, (_a: any, b: any) => {
+      const r = item.propertyMap[b]
+      if (typeof r === 'object') {
+        return r.join(', ')
       }
-      if (ips.length < 1 && str === '{id}' && item.isNode) {
-        ips = '<id>'
+      if (typeof r === 'string' || typeof r === 'number') {
+        return r
       }
-      return ips.replace(/^<(id|type)>$/, (_a: any, b: any) => {
-        const r = item[b]
-        if (typeof r === 'string' || typeof r === 'number') {
-          return r
-        }
-        return ''
-      })
-    }
-
-    GraphStyle.prototype.forNode = function(node: any = {}) {
-      const selector = nodeSelector(node)
-      if ((node.labels != null ? node.labels.length : 0) > 0) {
-        this.setDefaultNodeStyling(selector, node)
+      return ''
+    })
+    if (ips.length < 1 && str === '{type}' && item.isRelationship) {
+      ips = '<type>'
+    }
+    if (ips.length < 1 && str === '{id}' && item.isNode) {
+      ips = '<id>'
+    }
+    return ips.replace(/^<(id|type)>$/, (_a: any, b: any) => {
+      const r = item[b]
+      if (typeof r === 'string' || typeof r === 'number') {
+        return r
       }
-      return this.calculateStyle(selector)
-    }
+      return ''
+    })
+  }
 
-    GraphStyle.prototype.forRelationship = function(rel: any) {
-      const selector = relationshipSelector(rel)
-      return this.calculateStyle(selector)
+  forNode = (node: any = {}): StyleElement => {
+    const selector = this.nodeSelector(node)
+    if ((node.labels != null ? node.labels.length : 0) > 0) {
+      this.setDefaultNodeStyling(selector, node)
     }
+    return this.calculateStyle(selector)
+  }
 
-    return GraphStyle
-  })()
-  // @ts-expect-error ts-migrate(7009) FIXME: 'new' expression, whose target lacks a construct s... Remove this comment to see the full error message
-  return new GraphStyle()
+  forRelationship = (rel: any): StyleElement => {
+    const selector = this.relationshipSelector(rel)
+    return this.calculateStyle(selector)
+  }
 }
+
+export default GraphStyle
```

Comment:
Should this file be names as `GraphStyle.ts` as it's a class/prototype? Same as `node.ts` to `Node.ts`, `relationship.ts` to `Relationship.ts`. WDYD?
Also, I'm not sure why this file is placed in the root folder. It should go with `/components` or `/renders`?

Reply:
Yes, probably, but I leave it as it is now and we can change the file structure in another PR, since for example graphStyle contains another class Selector, which also is exported. Probably I should move this out of this file as well then.

Reply:
Agree!